src/viewer.js

```diff
@@ -1227,8 +1227,8 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
      * @param {OpenSeadragon.Rect} [options.clip] - An area, in image pixels, to clip to
      * (portions of the image outside of this area will not be visible). Only works on
      * browsers that support the HTML5 canvas.
-     * @param {Number} [options.opacity=1] Opacity the tiled image should be drawn at by default.
-     * @param {Boolean} [options.preload=false]  If true, tiles in image load without drawing by default.
+     * @param {Number} [options.opacity=1] Default proportional opacity of the tiled images (1=opaque, 0=hidden)
```

Comment:
I don't know that these comments need to say "Default", since it's just for the one image that's being added. If anything it could be "initial", but even that is probably unnecessary.